### PR TITLE
refactor(cdn.dedicated): refactor cdn manage statistics

### PR DESCRIPTION
**Pull request by JeremyDec**
## Refactor of CDN Dedicated statistics page
Following MBP-183 resolution
### Description of the Change
Here comes a refactor of the code, of cdn > dedicated > manage > statistics part.

(This needs that controllers of modal actions to use $uibModal)

### CODE REVIEW

File: client/app/cdn/dedicated/domain/add/cdn-dedicated-domain-add.html [hunk 1]
```diff
@@ -1,91 +1,87 @@
-<div data-ng-controller="CdnAddDomainsCtrl">
+<div data-wizard
```
Comment (JeremyDec) [inline client/app/cdn/dedicated/domain/add/cdn-dedicated-domain-add.html]: Only previous inline `data-ng-controller` has been changed in this file (+ adding $uibModalInstance.dismiss)

File: client/app/cdn/dedicated/domain/rule/order/cdn-dedicated-domain-rule-order.html [hunk 2]
```diff
@@ -1,133 +1,129 @@
...
-
-        <div data-wizard-step
-             data-wizard-step-on-load="loadCacherulesPrice"
-             data-wizard-step-valid="choices.count">
-
-            <div class="text-center"
-                 data-ng-if="cacheRulesLoading">
-                <oui-spinner></oui-spinner>
-            </div>
+<div data-wizard
```
Comment (JeremyDec) [inline client/app/cdn/dedicated/domain/rule/order/cdn-dedicated-domain-rule-order.html]: Only previous inline `data-ng-controller` has been changed in this file (+ adding $uibModalInstance.dismiss)

File: client/app/cdn/dedicated/quota/order/cdn-dedicated-quota-order.html [hunk 3]
```diff
@@ -1,101 +1,97 @@
-<div data-ng-controller="OrderQuotaCtrl">
+<div data-wizard
```
Comment (JeremyDec) [inline client/app/cdn/dedicated/quota/order/cdn-dedicated-quota-order.html]: Only previous inline `data-ng-controller` has been changed in this file (+ adding $uibModalInstance.dismiss)

File: client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.component.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import template from './cdn-dedicated-manage-statistics.html';
+
+angular.module('App').component('cdnStatistics', {
+  template,
+  controller: 'CdnStatisticsCtrl',
```
Comment (marie-j) [inline client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.component.js]: Why not `export` class and just `import controller from './controller` like you did for `template` ?

File: client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.controller.js [hunk 5]
```diff
@@ -1,53 +1,96 @@
...
-      .then(data => createChart(data))
-      .catch((err) => {
-        if (err.message) {
-          _.set(err, 'message', err.message.replace(' : null', ''));
-          $scope.setMessage($translate.instant('cdn_configuration_add_ssl_get_error'), { type: 'ERROR', message: err.message });
+import addDomainTemplate from '../../domain/add/cdn-dedicated-domain-add.html';
+import orderRuleTemplate from '../../domain/rule/order/cdn-dedicated-domain-rule-order.html';
+import orderQuotaTemplate from '../../quota/order/cdn-dedicated-quota-order.html';
+
+class CdnStatisticsCtrl {
```
Comment (marie-j) [inline client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.controller.js]:
```suggestion
export default class CdnStatisticsCtrl {
```

File: client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.component.js (hunk 4 above)
Comment (JeremyDec) [reply client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.component.js]: Yes I could've do it, but this implies to create a `index.js` (placed in grand-parent directory) that list all `import controller from './controller'`, and with this file created, this also implies to refactor parent controllers etc etc...

So I've just prepared the path to `export`/`import`, (with the creation of the angularjs controller placed at the bottom of https://github.com/ovh-ux/ovh-manager-dedicated/pull/1006/files/448ad3d4d0c3696fa60ecc7ace153f6c20628010#diff-8b8547fec71a561e51a97106c23b8f27 file)

File: client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.controller.js (hunk 5 above)
Comment (JeremyDec) [reply client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.controller.js]: See https://github.com/ovh-ux/ovh-manager-dedicated/pull/1006#discussion_r264135663

**Comment by jleveugle** [comment]
Hello @JeremyDec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**